src/core/kmod-setup.c

```diff
@@ -45,6 +53,41 @@ static void systemd_kmod_log(
 }
 #endif
 
+static bool has_virtio_rng(void) {
+        char * const paths[] = { (char *)"/sys/devices/pci0000:00", NULL };
+        _cleanup_fts_close_ FTS *fts = NULL;
+
+        fts = fts_open(paths, FTS_NOCHDIR|FTS_NOSTAT, NULL);
```

Comment:
maybe use STRV_MAKE() here for the first argument? (possibly cast, regarding the const thing). That way we don't need any local variable for this